controller/scheduler/swarm.py

```diff
@@ -6,7 +6,6 @@
 
 from .states import JobState
 
-
```

Comment:
Two lines are required here as per @mboersma's comment about PEP8. It recommends one line between import groupings and two lines separating class definitions, variables, methods and import statements from each other. I think we skip the scheduler package entirely from `make test-style` due to the longevitiy of some lines (the container templates in fleet.py, more specifically) but it's still good to follow best practices whenever possible.